### DISCUSSION TITLE
Translucent not working

### DISCUSSION THREAD

**Discussion by OfficialGuru** [Akifyss/obsidian-border#241] [Q&A] [answered]
Followed instructions and enabled translucent in settings and translucency is not enabled.

![CleanShot 2024-04-17 at 20 00 31@2x](https://github.com/Akifyss/obsidian-border/assets/42572944/109f6846-189a-4ad4-91cb-768a4c79c4cf)

**Comment by Akifyss** [comment]
Is your system running on Windows or Linux? Recently, the official Obsidian team removed the translucency option, which might be related to this. Currently, only Mac OS should be able to use translucency.

https://obsidian.md/changelog/2024-03-19-desktop-v1.5.11/

![image](https://github.com/Akifyss/obsidian-border/assets/23567154/9420f7d5-8301-41a0-b2a0-45d77ea871a6)

**Reply by OfficialGuru** [reply]
@Akifyss  Currently using Mac OS, so it should be working.

**Reply by Akifyss** [reply, marked as answer]
I tested it on macOS, it works, but I encountered a bug, in some cases, I need to first maximize the window for the translucency effect to appear:
<img width="1024" alt="image" src="https://github.com/Akifyss/obsidian-border/assets/23567154/c5b277c9-6369-4081-9c1d-bc0af06c8021">
